forum/qa-content/qa-page.js

```diff
@@ -277,7 +277,7 @@ function qa_ajax_error()
         const questionId = parseInt(location.pathname.split('/')[1]);
         const newQuestion = location.pathname.includes('ask');
 
-        if (questionId || newQuestion) {
+        if (typeof CKEDITOR !== 'undefined' && (questionId || newQuestion)) {

```

Comment:
To sprawdzenie obecności CKEditora przeniósłbym na początek [funkcji `postPreview`](https://github.com/CodersCommunity/forum.pasja-informatyki.local/blob/ab9babf5517c6f5b0cdda6b5b3cfc20946d040b3/forum/qa-content/qa-page.js#L212-L215), bo po co w ogóle ją wykonywać, jeśli CKEditor nie jest dostępny na stronie.

Reply:
Ok, przeniesione